django/santropolFeast/order/models.py

```diff
@@ -108,6 +114,10 @@ def filter_search(self, queryset, value):
                     if user not in query:
                         query.append(user)
 
+                for user in status:
+                    if user not in query:
+                        query.append(user)
+
```

Comment:
You don't need to use custom filter for the status.
Just use this: https://github.com/savoirfairelinux/santropol-feast/blob/dev/django/santropolFeast/member/models.py#L332